main/scraping/gainers.py

```diff
@@ -0,0 +1,74 @@
+from bs4 import BeautifulSoup
+import pandas as pd
+
+def daily_gainers():
+    dfs = pd.read_html('https://money.rediff.com/gainers/bse/daily',header=0)
+    for df in dfs[:-1]:
+        df=df
+
+    df['% Change'] = df['% Change'].str.replace(' ', "")
+    df1 = df[['Company', '% Change','Current Price (Rs)']][:10]
+    data = {}
+    col = list(df1)
+    for i in range(10):
+        current = 'Company {}'.format(i+1)
+        data[current] = {}
+        c=0
+        for j in col:
+            if c==0:
+                data[current]['Company Name'] = df[j][i]
+            elif c==1:
+                data[current]['% Change'] = df[j][i]
+            else:
+                data[current]['Current Price (Rs)'] = df[j][i]
+            c+=1
+    
+    return data
+
```

Comment:
@hpb1  functions for calculating daily , weekly and monthly gainers have already been pushed by @Bhard27  under the scraping/gainers directory we do there seems to be no need to have the same code in two separate locations.

The only difference between both the codes is how the variable c is being updated each case
`else:
                data[current]['Current Price (Rs)'] = df[j][i]
  c+=1 
`
please take a look into this